BrainPortal/app/controllers/portal_controller.rb

```diff
@@ -361,6 +363,23 @@ def search
     @results = @search.present? ? ModelsReport.search_for_token(@search, current_user) : {}
   end
 
+  # A HTML GET request produces a SwaggerUI information page.
+  # A JSON GET request sends the swagger specification.
+  def swagger
+    # Find latest JSON swagger spec.
+    # FIXME sort() will break when comparing versions...
```

Comment:
Not very important right now, until we have many many API files to distribute... and right now we're still working on writing the first one!